base/v0_2_exp/translate.go

```diff
@@ -88,6 +95,23 @@ func translateIgnition(from Ignition, options base.TranslateOptions) (to types.I
 	return
 }
 
+func translateStorage(from Storage, options base.TranslateOptions) (to types.Storage, tm translate.TranslationSet, r report.Report) {
```

Comment:
It doesn't feel great that we have to define a custom translator to just ignore a field that only exists in one struct. I get that we don't want to automatically silently ignore fields but maybe we can add another mechanism to explicitly say "we're handling this field elsewhere, please ignore it" when the general translator is passing through?